outputs.tf

```diff
@@ -0,0 +1,22 @@
+# ------------------------------------------------------------------------------
+# Output
+# ------------------------------------------------------------------------------
+output "service_arn" {
```

Comment:
We have started adding `description = "<something>"` to output in other modules, because it renders more nicely when we publish it. E.g.:  
https://registry.terraform.io/modules/telia-oss/asg/aws/0.2.0?tab=outputs

I think you could mostly copy-paste descriptions from here:
https://github.com/telia-oss/terraform-aws-ecs/blob/master/modules/service/outputs.tf
(Which are mostly copy-pasted themselves from the terraform documentation)